tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
```

Comment:
We try to avoid adding tests/changes to tests/unittests/Foundation/**ReferenceFoundation/**
If you want to add tests they should go in the tests/unittests/**Foundation** folder. The following is the file name convention we used for tests that we created NS*Tests.mm. The imports from swift are named TestNS*.mm